docs/src/main/sphinx/sql/show-functions.rst

```diff
@@ -12,10 +12,32 @@ Synopsis
 Description
 -----------
 
-List all the functions available for use in queries.
+List all the functions available for use in queries. For each function returned,
+the following information is displayed:
+
+* Function name
+* Return type
+* Argument types
+* Function type
+* Deterministic
+* Description
 
 :ref:`Specify a pattern <like_operator>` in the optional ``LIKE`` clause to
 filter the results to the desired subset. For example, the following query
 allows you to find functions beginning with ``array``::
 
     SHOW FUNCTIONS LIKE 'array%';
+
+``SHOW FUNCTIONS`` works with built-in functions as well as with :doc:`custom
+functions </develop/functions>`. In the following example, three custom
+functions with the ``cf_`` prefix are available:
+
+.. code-block:: text
+
+   SHOW FUNCTIONS LIKE 'cf_%';
```

Comment:
This explanation is misleading because an underscore in LIKE means any single character. I would recommend rewriting this example.

Reply:
+1

Reply:
After some study, I now have more concerns about the `'cf_%'` example than just the confusion about the role of the underline. To wit: where did those `cf_*` functions come from? There are not in the standard set of built-in functions for SEP 375-e running locally or on our Demo Docs cluster, and no function beginning with `cf` appears in our alpha list of functions in the docs.

Idly watching TV and trying different things, I came up with the following example commands that produce only two or three lines of output and actually work in 375-e:

```
SHOW FUNCTIONS like 'a__n'
SHOW FUNCTIONS like  'avg'
SHOW FUNCTIONS like '__rt'
SHOW FUNCTIONS like 'len%'
SHOW FUNCTIONS like 'log%'
SHOW FUNCTIONS like 'yo%'
SHOW FUNCTIONS like 'spook%'
```

Tip: in our query editor, with the results still on the screen, click Download. This saves a CSV file locally. If you have Apple's default and free Office products installed, CSV files open in Numbers, making a beautiful result, like this one that can be screen-shot and put in the docs without talking up too much room:

![image](https://user-images.githubusercontent.com/5950884/163253181-b868b598-0fea-4f15-808c-82f0252f086b.png)